src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,29 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.12.0
+29 August 2023
+
+This version is compatible with Kubernetes versions 1.24 to 1.26.
+
+### Breaking changes
+
+- Kubeconfigs downloaded from the Okteto web UI now require Okteto CLI 2.15 or higher to use dynamic tokens for authenticating against Kubernetes. <!-- 6972 -->
+
+### Features
+
+- The Okteto CLI has been upgraded to version [2.19.2](https://github.com/okteto/okteto/releases/tag/2.19.2).
+- The Admin view has been redesigned with a new vertical tab column. <!-- 6851 -->
+- Added "Registry Credentials" to the Admin view to display configured private registry information. This provides visibility into the private registries in use for your Okteto instance (username/password and AWS Access/Secret only for now) (link to docs TBC) <!-- 6851 -->
```

Comment:
We're missing the priority class feature https://github.com/okteto/app/commit/864297313e86c71f47613bfd5d2b1a7364d57332

I'll send another PR documenting it, and then you can link it here

Reply:
https://github.com/okteto/docs/pull/483